### ISSUE TITLE
[BUG] Math.Pow()

### ISSUE THREAD

**Issue by 365Tito** [CosmosOS/Cosmos#871]
Math.Pow(2,2) gives the result 3

```
Console.WriteLine("INIT. POWTEST");
 int x = (int)Math.Pow(2, 2);
Console.WriteLine("2 ** 2 = " + x.ToString());
Console.WriteLine("EXECUTED OK.");
```

result:

![screenshot 1](https://user-images.githubusercontent.com/29928707/35199023-0e6cd3bc-fef8-11e7-9853-cb1906a74e09.jpg)

Math.Pow(4,4) gives 255 etc...

**Comment by jp2masa** [comment]
Are you using the User Kit or the Dev Kit?

**Comment by 365Tito** [comment]
@jp2masa  I'm using the Dev Kit.

**Comment by jp2masa** [comment]
Are you using the latest Dev Kit?

**Comment by 365Tito** [comment]
The latest dev kit wont work... 
I programmed my own Power function for now so I guess I could go without it :)
Thanks anyways!

**Comment by jp2masa** [comment]
It should have been fixed by #835.
